Library/Source/NativeEngine.cpp

```diff
@@ -799,14 +858,26 @@ namespace babylon
         textureData->Images.push_back(bimg::imageParse(&m_allocator, buffer.Data(), static_cast<uint32_t>(buffer.ByteLength())));
         auto& image = *textureData->Images.front();
 
+        bool useMipMap = false;
+        auto imageDataRef{ bgfx::makeRef(image.m_data, image.m_size) };
+        if (mipMap)
+        {
+            auto imageDataRefMipMap = GenerateMipMaps(image);
+            if (imageDataRefMipMap)
```

Comment:
might be good to log something when mipmaps are not generated when they should

Reply:
No console/log function is accessible to the C++ for now. It's interesting to create a console object and log(const char*) function but not in the scope of this PR. Adding a comment for now.

Reply:
That'll likely be a part of the future change that will perpetuate the new pattern (used for NativeEngine and NativeWindow as of my inbound PR) to the other parts of the runtime-owned state. That change should make the console readily accessible from within any of our native JavaScript objects.